### PR TITLE
added loading animation in onboarding page

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

**Comment by yuda1124** [comment]
good. but if it is not just for login, how about change name?

**Comment by ochanje210** [comment]
it IS only for login tho. do you think it would be used else where?

File: shared/components/LoginFacebook.ios.js [hunk 1]
```diff
@@ -76,16 +76,19 @@ class LoginFacebook extends Component {
...
         setLoginType('facebook');
         grantFacebookUser(data.credentials.token).then(() => {
           this.props.setToken('facebook');
           console.log(this.props.currentScene);
+          this.props.setLoadingLoginAnimating(false);
           if (this.props.currentScene === 'initialScene') {
             this.props.setCurrentScene('map');
             Actions.map({type: 'replace'});
           }
         });
```
Comment (ovekyc) [inline shared/components/LoginFacebook.ios.js]: you can return inside the if statement and remove else statement

**Comment by ovekyc** [comment]
lgtm

**Pull request merged**